play/objects/image.py

```diff
@@ -40,12 +40,13 @@ def update(self):
             self.rect = draw_image.get_rect()
             pos = convert_pos(self.x, self.y)
             self.rect.center = pos
+            self._transformed_image = draw_image
             super().update()
 
     @property
     def image(self):
         """Return the image."""
-        return self._image
+        return getattr(self, "_transformed_image", self._image)
```

Comment:
```suggestion
        return self._transformed_image if self._transformed_image else self._image
```

getattr() wordt gebruikt als het onbekend is of de member/property aanwezig is, maar dat is hier niet het geval. Beter om dan geen getattr() te gebruiken: veel IDE's missen dit als property access bij bijv. een refactor/rename.